Source/Csla.Blazor/State/StateManager.cs

```diff
@@ -57,12 +56,12 @@ private async Task GetState(TimeSpan timeout)
     /// at which you know the user is navigating to another
     /// page.
     /// </remarks>
-    public void SaveState()
+    public void SaveState(TimeSpan timeout)
```

Comment:
@rockfordlhotka Is there any reason why this isn't `Task` too? Did you want to avoid `IAsyncDisposable` or just fire&forget for the user? Writing this it's awkward to have a timeout but not the choice to wait for the timeout or not 🤔

Reply:
I'm open to get another tasks and update the async, maybe start adding cancellation token in other interfaces

Reply:
I don't know about adding cancellation token in other interfaces.
I don't think it's easy to add it easily in the whole framework. We should definitly consult @rockfordlhotka before anyone tackles that.

Reply:
Do you want to change this to a Task now?

Reply:
It should return `Task`. 

I am fine with an overload of the method that optionally accepts a cancelation token - assuming it is possible to cancel the underlying operation.

On the server, the operation typically isn't even async, but _could_ be if someone used REDIS or something. On the wasm client it uses http, and so we may be able to pass the token to the `HttpClient` call?